code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -37,7 +37,7 @@ emp_act
 		return 2
 
 	if(mind?.martial_art?.deflection_chance) //Some martial arts users can deflect projectiles!
-		if(!lying && !HAS_TRAIT(src, TRAIT_HULK) && prob(mind.martial_art.deflection_chance)) //But only if they're not lying down, and hulks can't do it
+		if(!lying && !HAS_TRAIT(src, TRAIT_HULK) && !HasDisease(new /datum/disease/memetic_madness(0)) && prob(mind.martial_art.deflection_chance)) //But only if they're not lying down, and hulks can't do it
```

Comment:
My god, screw whoever wrote the `HasDisease()` code and made it so it takes an instance of a disease instead of just a type.